incubator/monochart/templates/_helpers.tpl

```diff
@@ -99,3 +99,14 @@ VolumeMounts template block for deployable resources
 {{- end }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Image template block
+*/}}
+{{- define "monochart.pod.image" -}}
+{{- $global := (index . 0).Values.image | default dict -}}
+{{- $override := (index . 1).image | default dict -}}
```

Comment:
where this image `(index . 1).image` coming from?

Reply:
https://github.com/cloudposse/charts/pull/188/files#diff-01b89b0376166b7fc10166b6a89c0018R43
https://github.com/cloudposse/charts/pull/188/files#diff-d9bcae96f38b129da185e3eb7ef3a753R43
https://github.com/cloudposse/charts/pull/188/files#diff-360524444cea4b55d5d8e216495cf3edR49
https://github.com/cloudposse/charts/pull/188/files#diff-e040bfec97698f941d13388ededc9e0eR37
https://github.com/cloudposse/charts/pull/188/files#diff-019aa2f911ddd1cb19cabafb9806ca9eR46

it possible to override it in following way
```yaml
deployment:
  image:
    ...
```